tests/designs/array_module/Makefile

```
@@ -54,5 +54,5 @@ else
     $(error "A valid value (verilog or vhdl) was not provided for TOPLEVEL_LANG=$(TOPLEVEL_LANG)")
 endif
 
-include $(COCOTB)/makefiles/Makefile.inc
-include $(COCOTB)/makefiles/Makefile.sim
+include $(shell cocotb-config --makefiles)/Makefile.inc
+include $(shell cocotb-config --makefiles)/Makefile.sim
```

Could halve the number of calls and use:
```suggestion
include $(shell cocotb-config --makefiles)/Makefile.sim
include $(COCOTB_SHARE_DIR)/makefiles/Makefile.sim
```
@themperek, thoughts?

---

Yes but better to remove `include $(shell cocotb-config --makefiles)/Makefile.inc` than half solutions?

---

Right, I'd forgotten the goal eventually is to remove the first file entirely.